pkg/testutil/objstore.go

```diff
@@ -21,29 +21,35 @@ import (
 //
 // TODO(fabxc): define object storage interface and have this method return
 // mocks or actual remote buckets depending on env vars.
-func NewObjectStoreBucket(t testing.TB) (*gcs.Bucket, func()) {
+func NewObjectStoreBucket(t testing.TB) (*gcs.Bucket, func(), error) {
```

Comment:
What's the reason for this change? It seems to add a lot of boiler plate to the tests.

Reply:
I would rather prefer boilerplate, than being unable to cleanly return. It helps to have clear "cleanup path". If we use "cool" `testuitl.Ok(t, err)` there is no way to cleanly close/cleanup something that was allocated already. 

Otherwise we can handle this by just using `t.FailNow` + `fmt.Print` but it is just worse then `return nil, err`

And IMO having
 ```
bkt := testutilNewObjectStoreBucket
```
vs
```
bkt, err := testutilNewObjectStoreBucket
testuitl.Ok(t, err)
```
is really same. Not much of boilerplate in exchange for less prone to bug code. What do you think?

Reply:
Thanks for explaining, that makes sense.